blocks/key-facts/key-facts.css

```diff
@@ -2,138 +2,175 @@
   text-align: center;
 }
 
-.key-facts-wrapper .key-facts {
-  padding-top: 24px;
-  padding-bottom: 24px;
+.key-facts {
+  padding: 8px 0;
+  display: flex;
+  flex-direction: column;
+  gap: 24px;
 }
 
-.key-facts-wrapper .key-facts > div {
+.key-facts__row {
   display: flex;
-  flex-wrap: wrap;
   justify-content: center;
-  gap: 0 30px;
+  padding: 0;
+  flex-flow: row wrap;
+  row-gap: 32px;
+  column-gap: 8px;
 }
 
-.key-facts-wrapper .key-fact {
+.key-facts__key-item {
   display: flex;
-  flex-flow: column;
-  flex: 0 0 100%;
-  align-items: center;
-  padding: 0 15px;
-  margin-bottom: 32px;
+  flex-direction: column;
+  gap: 8px;
+  width: calc(50% - 4px);
+  justify-content: center;
 }
 
-.key-facts-wrapper .key-fact p {
+.key-facts__key-item p {
   margin: 0;
-  text-align: center;
 }
 
-/* icon */
-.key-facts-wrapper .key-fact i.fa {
-  font-size: 30px;
-  margin-bottom: 8px;
-}
-
-.key-facts-wrapper .key-fact .icon,
-.key-facts-wrapper .key-fact .icon svg {
-  height: 48px;
-  width: 48px;
+.key-facts .key-facts__main-text-wrapper {
```

Comment:
Could you add `text-wrap: balance;`? when the text is long, looks better ;)

Reply:
Nice, done